LoopKitUI/View Controllers/DismissibleHostingController.swift

```diff
@@ -8,22 +8,56 @@
 
 import SwiftUI
 
-public class DismissibleHostingController: UIHostingController<AnyView> {
+public struct _DismissibleHostingView<Content: View>: View {
```

Comment:
What is the underscore communicating here? It's not private, as the type is marked public.

Reply:
This follows a convention used by Apple for code that must be declared public (it's used in the type declaration of a public function) but should not be used without caution. The DismissibleHostingController initializer will wrap it's content in this view.